extension/task/utils/getSharedVariables.ts

```diff
@@ -7,6 +7,9 @@ import getDockerImageTag from "./getDockerImageTag";
 import getGithubAccessToken from "./getGithubAccessToken";
 
 export interface ISharedVariables {
+  /** Job ID */
+  jobId: string;
+
   /** URL of the organization. This may lack the project name */
```

Comment:
I do not see the value in making this configurable.

Reply:
Removed.